lib/lita/handlers/artifactory.rb

```diff
@@ -50,6 +50,18 @@ def promote(response)
           return
         end
 
+        # Validate the artifacts all exist in `omnibus-current-local`
+        unless repos_for_build(build).all? { |r| r == 'omnibus-current-local' }
+          reply_msg = <<-EOH.gsub(/^ {12}/, '')
+            :hankey: *#{project}* *#{version}* does not exist in the current channel.
+
+            The *#{project}* *#{version}* build did not pass required testing in its respective pipeline and was not automatically promoted from the _unstable_ to _current_ channel.
```

Comment:
Maybe turn this around? e.g.  "The *#{project}* *#{version}* build was not promoted from _unstable_ to _current_ because it did not pass the testing stage of its pipeline."